devicetypes/smartthings/inovelli-dimmer.src/inovelli-dimmer.groovy

```diff
@@ -0,0 +1,606 @@
+			range          : "0..10",
+			description    : "This will set the intensity of the LED bar (ie: how bright it is). Example of how the values work: 0 = Off, 1 = Low, 5 = Medium, 10 = High. This parameter can be set without a HUB from the Configuration Button."
+		],
+		[
+			name           : "LED Indicator Intensity (When Off)", key: "ledIndicatorIntensity(WhenOff)", type: "range",
+			parameterNumber: 15, size: 1, defaultValue: 1,
+			range          : "0..10",
+			description    : "This is the intensity of the LED bar when the switch is off. Example of how the values work: 0 = Off, 1 = Low, 5 = Medium, 10 = High. This parameter can be set without a HUB from the Configuration Button."
+		],
+		[
+			name           : "LED Indicator Timeout", key: "ledIndicatorTimeout", type: "range",
+			parameterNumber: 17, size: 1, defaultValue: 3,
+			range          : "0..10",
+			description    : "Changes the amount of time the RGB Bar shows the Dim level if the LED Bar has been disabled. Example of how the values work: 0 = Always off, 1 = 1 second after level is adjusted, 10 = 10 seconds after level is adjusted."
+		],
+		[
+			name           : "Active Power Alerts", key: "activePowerAlerts", type: "boolRange",
+			parameterNumber: 18, size: 1, defaultValue: 10,
+			range          : "1..100", disableValue: 0,
+			description    : "The power level change that will result in a new power report being sent (% of previous report). Example of how the values work: 0 = Disabled, 10 = 10% of previous report, 100 = 100% of previous report."
+		],
+		[
+			name           : "Periodic Power + Energy Reports", key: "periodicPower+EnergyReports", type: "range",
+			parameterNumber: 19, size: 2, defaultValue: 3600,
+			range          : "0..32767",
+			description    : "Time period between consecutive power and energy reports being sent (in seconds). Example of how the values work: 0 = 0 seconds, 1 = 1 second, 32767 = 32767 seconds. The timer resets after every report is sent."
+		],
+		[
+			name           : "Dimming Speed (Z-Wave)", key: "dimmingSpeed(Z-Wave)", type: "range",
+			parameterNumber: 2, size: 1, defaultValue: 101,
+			range          : "0..101",
+			description    : "How fast or slow the light turns dim when you adjust the switch remotely (ie: dimming from 10-20%, 80-60%, etc). Example of how the values work: 0 = Instant On, 1 = 1 second, 100 = 100 seconds. Entering the value of 101 = Keeps the switch in sync with Parameter 1."
+		],
+		[
+			name           : "Energy Reports", key: "energyReports", type: "boolRange",
+			parameterNumber: 20, size: 1, defaultValue: 10,
+			range          : "1..100", disableValue: 0,
+			description    : "The energy level change that will result in a new energy report being sent (% of previous report). Example of how the values work: 0 = Disabled, 10 = 10% of previous report, 100 = 100% of previous report."
+		],
+		[
+			name           : "Ramp Rate", key: "rampRate", type: "range",
```

Comment:
what's the difference between ramp rate and dimming speed?

Reply:
Dimming speed is time when lights dim when button is pressed
Ramp rate is time when button is clicked and state change is on/off